MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/extrusions/AdjustExtrusionLightActivity.java

```diff
@@ -60,8 +61,13 @@ protected void onCreate(Bundle savedInstanceState) {
       @Override
       public void onMapReady(@NonNull final MapboxMap map) {
         mapboxMap = map;
-        setupBuildings();
-        setupLight();
+        map.setStyle(Style.DARK, new Style.OnStyleLoaded() {
+          @Override
+          public void onStyleLoaded(@NonNull Style style) {
+            setupBuildings();
+            setupLight();
```

Comment:
pass Style as parameter to above 2 methods

Reply:
fixed